package.json

```diff
@@ -44,14 +44,51 @@
     "pnpm": ">=8.6.12"
   },
   "packageManager": "pnpm@8.6.12",
+  "dependencies": {
+    "astro-benchmark": "workspace:*"
+  },
+  "devDependencies": {
```

Comment:
took me a second and I thought you were adding all of these for the first time :D

Reply:
Ah, apologies, just organizing a little bit!